src/components/SignUpProfile/SignUpProfile.jsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react"
+import { useState } from "react"
+import { Link, useNavigate } from "react-router-dom"
+import { Button, Card, Alert } from "react-bootstrap" 
+import { doc, updateDoc } from "firebase/firestore"
+import { useAuth } from "../../contexts/AuthContext"
+import { db } from "../../firebase"
+import "./SignUpProfile.css"
+
+export default function SignUpProfile(props) {
+    // get user data from the database
+    const { currentUser } = useAuth()
+    const navigate = useNavigate()
+    const [loading, setLoading] = useState(false)
+    const diets = ["Gluten Free", "Ketogenic", "Vegetarian", "Vegan", "Pescetarian", "Paleo"]
```

Comment:
You may want to consider having these loaded from the database in the future so you can add/remove options more easily and keep them in sync with the data in the backend.